src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  box-sizing: border-box;
+  padding: 50px 40px;
+  display: grid;
+  place-content: center;
+  place-items: center;
+  row-gap: 48px 46px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  transition: all 300ms ease;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 0;
+  }
+
+  &__item {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-bottom: 40px;
+
+    background-size: contain;
+    background-image: url(../images/imac.jpeg);
```

Comment:
The background image is hardcoded in the SCSS, which reduces flexibility. If this image is meant to change dynamically (for different products, for example), consider adding it through inline styles or classes in the HTML instead.